package.json

```diff
@@ -26,19 +27,22 @@
     "test:node": "(cd packages/node && yarn run test)",
     "test:node:watch": "(cd packages/node && yarn run test:watch)"
   },
+  "dependencies": {
+    "@types/node": "^14.10.2",
+    "lerna": "^3.22.1",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.0.2"
+  },
```

Comment:
I had to define all dependencies that I need to build the node inside the deployer container as non-development dependencies because I'm using `NODE_ENV`-`production`. This is probably not a good idea. Let me know what you think.

Reply:
This also happens in packages btw.

Reply:
Okay then. I think this is not a good enough reason to use global packages.